packages/desktop-client/src/components/Notifications.tsx

```diff
@@ -107,7 +107,11 @@ function Notification({
     <View
       style={{
         marginTop: 10,
-        color: positive ? colors.g3 : error ? colors.r3 : colors.y2,
+        color: positive
+          ? theme.noticeText
+          : error
+          ? theme.errorText
+          : theme.warningText,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏different colors used